backend/api/views/project.py

```diff
@@ -17,7 +18,10 @@ def get_queryset(self):
         return self.request.user.projects
 
     def perform_create(self, serializer):
-        serializer.save(users=[self.request.user])
+        if self.request.user.is_staff:
+            serializer.save(users=[self.request.user])
+        else:
+            raise ProjectCreationPermissionDenied()
```

Comment:
Maybe non-staff user but project admin can delete projects.
Defining staff permission and setting it to `permission_classes` may be a safer way.

Reply:
Thanks for your comments.
Changed to use `permission_classes`. https://github.com/doccano/doccano/pull/1453/commits/ed37696a67a0e1dc90fa5fdcd0c566e223a7b6c0